ckanext/gitdatahub/plugin.py

```diff
@@ -65,6 +70,21 @@ def after_update(self, context, pkg_dict):
                 json.dumps(body, indent=2),
                 contents.sha
                 )
+            resources_list = []
+            for obj in body['resources']:
+                resources_list.append(obj['title'])
+            
+            gitattributes_body = ''
+            for obj in resources_list:
+                gitattributes_body += "data/{} filter=lfs diff=lfs merge=lfs -text\n".format(obj)
```

Comment:
Just one minor change, there's no need for two foor loops, one should be enough. Something like this:
```python
gitattributes_body = ''
for obj in body['resources']:
    gitattributes_body += "data/{} filter=lfs diff=lfs merge=lfs -text\n".format(obj['title'])
```